src/components/inline-inputs/inline-inputs.component.tsx

```diff
@@ -25,7 +27,8 @@ type GutterOptions =
   | "extra-large";
 
 export interface InlineInputsProps
-  extends StyledContentContainerProps,
+  extends MarginProps,
```

Comment:
this seems to be a big change from the last version of this PR, in allowing margin props to be passed to InlineInputs. I don't mind it - I see no good reason not to allow consumers to set their own margin if they want - but we should make sure we include the margin props table in the docs. And probably make this part a separate "feat" commit, but that's not critical so I'll leave that to your judgement.

Reply:
Yeah good point I've missed the prop table, I'll be making this change a `feat` one I just wanted to make it easy for the updates to be reviewed for now

Reply:
Moved this to a separate `feat` commit that adds the interface and util. In the subsequent `fix` commit I refactor it to use the custom form spacing hook